rust/azure_iot_operations_services/src/azure_device_registry/client.rs

```diff
@@ -466,11 +466,11 @@ where
                                 Ok(()) => {
                                     log::debug!("Device Update Notification dispatched for device {device_name:?} and inbound endpoint {inbound_endpoint_name:?}");
                                 }
-                                Err(DispatchError::SendError(tokio::sync::mpsc::error::SendError((payload, _)))) => {
-                                    log::warn!("Device Update Observation has been dropped. Received Device Update Notification: {payload:#?}");
+                                Err(DispatchError { data, kind: DispatchErrorKind::SendError }) => {
```

Comment:
since before we were only printing the payload and not the ack token, would we want to keep that functionality? Might be something like this? Not sure on the compiler syntax
```suggestion
                                Err(DispatchError { (payload, _), kind: DispatchErrorKind::SendError }) => {
```
here and for the other 3 spots in this file